scalereg/reg6/staff/views.py

```diff
@@ -163,7 +172,7 @@ def CashPayment(request):
   attendee.email = request.POST['EMAIL']
   attendee.valid = True
   attendee.checked_in = True
-  attendee.can_email = True
+  attendee.can_email = False
```

Comment:
is this because most dont provie emails?

Reply:
That, and they did not give explicit consent.